entity-management-definitions/src/main/java/eu/europeana/entitymanagement/definitions/exceptions/EntityFieldAccessException.java

```diff
@@ -0,0 +1,25 @@
+package eu.europeana.entitymanagement.definitions.exceptions;
+
+/**
+ * This class is used represent validation errors for the grouping class hierarchy
+ *  
+ * @author GrafR 
+ *
+ */
+public class EntityFieldAccessException extends RuntimeException {
```

Comment:
By convention, all custom exceptions should extend `EuropeanaApiException`, unless there's a specific reason for this exception to be unchecked.

Reply:
yes, this is an exception that wrapps runtime exceptions that should not occur, like IllegalAccess. This class was simply renamed from EntityValidationException because of the naming collision with the Exception that indicates the real validation constraints.  That correct exception class extends the EuropeanaApiException